examples/logging/langchain_logger.py

```diff
@@ -0,0 +1,23 @@
+from langchain.chat_models import ChatAnthropic
```

Comment:
Consider naming this file `langchain_model_logger.py` since we have other langchain examples.
Add the new examples (incl the Anthropic ones) to the `examples/README`

Reply:
Will do!